### PR TITLE
revert creation takes full object change

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md [hunk 1]
```diff
@@ -4,9 +4,6 @@
 
 **Breaking Changes**
 
-- `create_data_feed` now takes as input a `DataFeed` object
-- `create_metric_anomaly_detection_configuration` now takes as input an `AnomalyDetectionConfiguration` object
-- `create_anomaly_detection_configuration` now takes as input an `AnomalyAlertConfiguration` object
 - `create_hook` now takes as input an `EmailHook` or `WebHook`
```
Comment (kristapratico) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md:7]:
```suggestion
```

Comment (kristapratico) [reply sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md:7]: Actually will we make the change to create_hook as well? It used to take the name parameter separate from the object.

Comment (xiangyan99) [reply sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md:7]: It seems to me the difference is
to take name+hook object -> to hook object only.

I don't think we should revert back to name+object.

File: sdk/metricsadvisor/azure-ai-metricsadvisor/tests/base_testcase.py [hunk 2]
```diff
@@ -177,48 +174,39 @@ def __init__(self, method_name):
...
+                    DataFeedDimension(name="city")
+                ],
+            ),
+            ingestion_settings="2019-10-01T00:00:00Z",
         )
 
     def _create_data_feed_and_detection_config(self, name):
         data_feed = self._create_data_feed(name)
-        detection_config_name = create_random_name(name)
+        detection_config_name = self.create_random_name(name)
```
Comment (kristapratico) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/tests/base_testcase.py:198]: no `self`

**Review by kristapratico — APPROVED** [review]

**Pull request merged**